.travis.yml

```diff
@@ -33,3 +33,16 @@ after_success:
   - bash <(curl -s https://codecov.io/bash)
   - cd $TRAVIS_BUILD_DIR/app
   - bash <(curl -s https://codecov.io/bash) -f coverage/coverage-final.json
+  - cd $TRAVIS_BUILD_DIR/service
+  - go run server.go 2> output.out &
```

Comment:
we should run the image instead of go file.

Reply:
Ok. I'll work on this to run the image instead of go file.

Reply:
@fzxu Thinking about if I can get this pr merge first, since this merge contains Dave's fix so travis will work without exit the build so that I don't have to make a work around to get rid of build failure. If this is ok, then I'll get latest merged code to work on to run docker image instead of go file. What do you think?

Reply:
@fzxu @formuzi Pls don't merge my pr request yet even though it looks like the checks are green. Since the commands in after_success section won't make checks fail status even though it fails. Looks like the build image for service has problem. I'm looking into it.

Reply:
@fzxu @formuzi It was a wrong alarm. Add image checks, the images were built successfully. Docker run image failed. Not sure if this command has any problem: $ docker run -d -p 8011:8011 katlas-service:${TRAVIS_COMMIT}